README.md

```
@@ -65,7 +61,7 @@ The workflow repository contains a small test dataset `.tests/integration` which
 
 ```bash
 $ cd .tests/integration
-$ snakemake -s ../../Snakefile -j1 --configfile config.yaml --use-singularity
+$ snakemake -s ../../workflow/Snakefile -j1 --configfile config.yaml --software-deployment-method apptainer
```

markdownlint-cli2 (0.23.2)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)